src/main/java/org/dependencytrack/policy/cel/persistence/CelPolicyVulnerabilityRowMapper.java

```diff
@@ -85,44 +76,32 @@ public Vulnerability map(final ResultSet rs, final StatementContext ctx) throws
         return builder.build();
     }
 
-    private void maybeSetAliases(final ResultSet rs, final Consumer<Iterable<Vulnerability.Alias>> setter) throws SQLException {
-        if (!hasColumn(rs, "aliases")) {
-            return;
-        }
-
-        final String aliasesJson = rs.getString("aliases");
-        if (aliasesJson == null) {
-            return;
-        }
-
-        try {
-            final List<Vulnerability.Alias> aliases = OBJECT_MAPPER.readValue(aliasesJson, new TypeReference<List<VulnerabilityAlias>>() {
-                    }).stream()
-                    .flatMap(alias -> alias.getAllBySource().entrySet().stream()
-                            .map(aliasEntry -> Vulnerability.Alias.newBuilder()
-                                    .setSource(aliasEntry.getKey().name())
-                                    .setId(aliasEntry.getValue())
-                                    .build()))
-                    .distinct()
-                    .toList();
-            setter.accept(aliases);
-        } catch (IOException e) {
-            throw new RuntimeException(e);
-        }
+    private static List<Integer> maybeConvertCwes(final ResultSet rs, final String columnName) throws SQLException {
+        return stringArray(rs, columnName).stream()
+                .map(Integer::parseInt)
+                .toList();
     }
 
-    private void maybeSetCwes(final ResultSet rs, final Consumer<Iterable<Integer>> setter) throws SQLException {
-        if (!hasColumn(rs, "cwes")) {
-            return;
+    private static List<Vulnerability.Alias> maybeConvertAliases(final ResultSet rs, final String columnName) throws SQLException {
+        final List<VulnerabilityAlias> aliases = json(rs, columnName, VULNERABILITY_ALIASES_TYPE_REF);
```

Comment:
can we rename this method to say convertJsonToGivenType or something?

Reply:
Renamed to `deserializeJson` to make it more clear what the method is doing.